### ISSUE TITLE
Almost all tests are broken

### ISSUE THREAD

**Issue by koddsson** [cloudflare/cf-ui#195]
### How to reproduce:

1. clone project
2. `npm run bootstrap`
3. `npm run build`
4. `npm run test`

### Expected behaviour: 

The tests run and come back passing

### Actual behaviour:

<details>
<summary>The tests run and all but one are failing.</summary>

```
...

 FAIL  packages/cf-component-modal/test/ModalBody.js
  ● Test suite failed to run

    /Users/koddsson/Repos/cf-ui/packages/cf-component-modal/test/ModalBody.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import React from 'react';
                                                                                             ^^^^^^
    SyntaxError: Unexpected token import

      at transformAndBuildScript (node_modules/jest-runtime/build/transform.js:320:12)
      at handle (node_modules/worker-farm/lib/child/index.js:41:8)
      at process.<anonymous> (node_modules/worker-farm/lib/child/index.js:47:3)
      at emitTwo (events.js:106:13)

 FAIL  packages/cf-builder-table/test/TableBuilder.js
  ● Test suite failed to run

    /Users/koddsson/Repos/cf-ui/packages/cf-builder-table/src/TableBuilder.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){var _createClass = function () {function defineProperties(target, props) {for (var i = 0; i < props.length; i++) {var descriptor = props[i];descriptor.enumerable = descriptor.enumerable || false;descriptor.configurable = true;if ("value" in descriptor) descriptor.writable = true;Object.defineProperty(target, descriptor.key, descriptor);}}return function (Constructor, protoProps, staticProps) {if (protoProps) defineProperties(Constructor.prototype, protoProps);if (staticProps) defineProperties(Constructor, staticProps);return Constructor;};}();function _classCallCheck(instance, Constructor) {if (!(instance instanceof Constructor)) {throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self, call) {if (!self) {throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call
    SyntaxError: Unexpected token import

      at transformAndBuildScript (node_modules/jest-runtime/build/transform.js:320:12)
      at Object.<anonymous> (packages/cf-builder-table/src/index.js:3:21)
      at Object.<anonymous> (packages/cf-builder-table/test/TableBuilder.js:7:14)

 FAIL  packages/cf-builder-pagination/test/PaginationBuilder.js
  ● Test suite failed to run

    /Users/koddsson/Repos/cf-ui/packages/cf-builder-pagination/src/PaginationBuilder.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){var _createClass = function () {function defineProperties(target, props) {for (var i = 0; i < props.length; i++) {var descriptor = props[i];descriptor.enumerable = descriptor.enumerable || false;descriptor.configurable = true;if ("value" in descriptor) descriptor.writable = true;Object.defineProperty(target, descriptor.key, descriptor);}}return function (Constructor, protoProps, staticProps) {if (protoProps) defineProperties(Constructor.prototype, protoProps);if (staticProps) defineProperties(Constructor, staticProps);return Constructor;};}();function _classCallCheck(instance, Constructor) {if (!(instance instanceof Constructor)) {throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self, call) {if (!self) {throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}r
    SyntaxError: Unexpected token import

      at transformAndBuildScript (node_modules/jest-runtime/build/transform.js:320:12)
      at Object.<anonymous> (packages/cf-builder-pagination/src/index.js:1:228)
      at Object.<anonymous> (packages/cf-builder-pagination/test/PaginationBuilder.js:4:14)


Test Suites: 105 failed, 1 passed, 106 total
Tests:       16 passed, 16 total
Snapshots:   0 total
Time:        10.783s
Ran all test suites.
```

</details>

**Comment by koddsson** [comment]
There's no documentation on how to run the tests so I assumed that the `npm run test` command would be sufficient.

**Comment by manatarms** [comment]
I followed your steps to reprod and all 106 tests failed. I'm assuming it's some secret step that we're missing? 
```
 FAIL  packages/cf-component-modal/test/ModalClose.js
  ● Test suite failed to run

    SyntaxError: Unexpected token [

      at transformAndBuildScript (node_modules/jest-runtime/build/transform.js:320:12)
      at Object.<anonymous> (test_setup/setup-fetch.js:1:118)


Test Suites: 106 failed, 106 total
Tests:       0 total
Snapshots:   0 total
Time:        4.299s
```

**Comment by koddsson** [comment]
Probably. I'm hoping someone will chime in with it so we can make a PR to document it and we can close this issue with that PR.

If it's not a known problem I can put on my debugging hat and see if I can figure out the issue 🤠

**Comment by hturan** [comment]
Yeah, this has always been the case for me, so +1.

**Comment by koddsson** [comment]
No one comment here so I'm gonna go in and see if I can figure this out.

**Comment by sejoker** [comment]
have you try using yarn to install dependencies? we have yarn.lock file in the repo

**Comment by koddsson** [comment]
Pretty sure. I'll double check right now.

**Comment by koddsson** [comment]
Yup. Still failing, did you see #196 @sejoker ?

**Comment by sejoker** [comment]
using yarn to install jest dependencies I don't see the same errors as you guys.

**Comment by koddsson** [comment]
Weird. It got fixed for me with #196 tho.
